README.md

```diff
@@ -38,7 +38,7 @@ This will drop you in GitHub's file editing interface, where you can
 modify the source code, preview it, and save your changes, by giving a
 short description of what you modified. If you have
 [write access](https://help.github.com/articles/what-are-the-different-access-permissions/)
-to the repository (hint: you do), your modifications will be published
```

Comment:
I am not sure what you mean by this. You do not have write access, but you can make a pull request ( as you did).

Reply:
The hint implies that everyone has write access, which is incorrect. I thought it should be removed.